packages/frontend/src/features/scheduler/components/SchedulerForm.tsx

```diff
@@ -1090,6 +1155,9 @@ const SchedulerForm: FC<Props> = ({
                                         </Text>
                                     )}
                                 </Stack>
+                                {health.data?.hasMicrosoftTeams && (
+                                    <MicrosoftTeamsDestination form={form} />
```

Comment:
maybe this should be set to `disabled` when `disabled` is true from the props in this component. 
Perhaps this is why the loading overlay doesn't "hide" that section. I could be wrong tho!

Reply:
you mean, rather than hiding the input ? yeah, maybe, but if we show it, people will start asking to enable this feature and I'd rather keep it private for now ..

Reply:
after doing https://github.com/lightdash/lightdash/pull/14254#discussion_r2026878173 ,  this got fixed 
![image](https://github.com/user-attachments/assets/6c6ca540-f217-47a2-a008-f6af39f846d1)